src/sentry/integrations/slack/message_builder/issues.py

```diff
@@ -224,20 +231,16 @@ def get_tags(
 
 def get_context(group: Group) -> str:
     context_text = ""
-    context = {
-        "Events": get_group_global_count(group),
-        "Users Affected": group.count_users_seen(),
-        "State": SUBSTATUS_TO_STR.get(group.substatus, "").replace("_", " ").title(),
-        "First Seen": time_since(group.first_seen),
-    }
-    if group.issue_type in REGRESSION_PERFORMANCE_ISSUE_TYPES:
-        # another short term solution for non-error issues notification content
-        return context_text
-
-    if group.issue_category in [GroupCategory.ERROR, GroupCategory.PERFORMANCE]:
-        for k, v in context.items():
-            if k and v:
-                context_text += f"{k}: *{v}*   "
+    context = group.issue_type.notification_config.context
+
+    context_dict = {}
+    for c in context:
+        if c in SUPPORTED_CONTEXT_DATA:
+            context_dict[c] = SUPPORTED_CONTEXT_DATA[c](group)
+
+    for k, v in context_dict.items():
+        if k and v:
+            context_text += f"{k}: *{v}*   "
```

Comment:
should we have the trailing space if it's the last key?

Reply:
i think there's some logic later to remove the trailing space. i could strip it here too

Reply:
yeah was just gonna say it's here https://github.com/getsentry/sentry/blob/0e1fbd5b7bf89f384b6fe984cf389843d464ae80/src/sentry/integrations/slack/message_builder/issues.py#L654 you could move it 🤷

Reply:
really nit picky but i think it makes sense to move it into here 😬